carbon-design-system/carbon
# 16879 detectoverflow configurable collision boundary

Part of #16879 

Added `hide` from Floating-ui to Dropdown component so it will hide the floating dropdown if the reference (Dropdown Trigger point) is not visible.


#### Testing / Reviewing

1. Check the ExperimentalAutoAlignBoundaryTest Story in Dropdown. 
2. Open the dropdown and scroll it, the moment 'Option 1' hide from the screen, it should hide the floating dropdown options as well.
